File: solutions/cpp/easy/c-tutorial-strings/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/c-tutorial-strings/problem?isFullScreen=true
// Problem     Strings
// Difficulty  Easy
// Subdomain   Strings
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-08-12, 09:56 a.m.
// ──────────────────────────────────────────────────

#include <iostream>
#include <string>
using namespace std;

int main() {
	string a;
    string b;
    
    cin >> a;
    cin >> b;
    
    cout << a.length() << " " << b.length() <<endl ;
    
    cout << a+b << endl;
    
    int temp=a.at(0);
    a.at(0)=b.at(0);
    b.at(0)=temp;
    
    cout << a << " " << b;
    
  
    return 0;
}

File: solutions/java/easy/java-substring/solution-approach1.java
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/java-substring/problem?isFullScreen=true
// Problem     Java Substring
// Difficulty  Easy
// Subdomain   Strings
// Platform    HackerRank
// Language    java
// Status      Accepted
// Submitted   2026-08-23, 11:47 a.m.
// ──────────────────────────────────────────────────

import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String S = in.next();
        int start = in.nextInt();
        int end = in.nextInt();
        
        String r=S.substring(start,end);
        System.out.println(r);
    }
}
